eks-distro-base/Makefile

```diff
@@ -7,17 +7,21 @@ BASE_IMAGE?=public.ecr.aws/amazonlinux/amazonlinux:2
 IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
 IMAGE_NAME?=eks-distro/base
 # This tag is overwritten in the prow job to point to the commit hash
-IMAGE_TAG?=$(shell git rev-parse HEAD | cut -c 1-8)
+IMAGE_TAG?=$(shell date "+%F-%s")
 IMAGE?=$(IMAGE_REPO)/$(IMAGE_NAME):$(IMAGE_TAG)
 
 ifeq ($(DEVELOPMENT),true)
 	BASE_IMAGE=amazonlinux:2
 endif
 
+ifeq ("$(REPO_OWNER)","")
+	$(error No org information was provided, please set REPO_OWNER environment variable. \
+	This is used to raise a pull request against your org after updating tags in the respective files.)
+endif
+
 .PHONY: local-images
-local-images:
-	# Sleep to wait for buildkitd to start up
-	sleep 10
```

Comment:
Let's keep the sleep as we aren't building binaries, unless we know that the script will take time to install dependencies before running any buildctl commands

Reply:
For the presubmits, the first buildctl command would be in the check-update script so I moved the sleep command there, while for postsubmits, I still have it as we run buildctl right off the bat.

Reply:
No let's move it back here so it remains consistent with images target below. It should be within the target instead of implicit in the script.

Reply:
Done